cypress/e2e/shared/tasks.test.ts

```diff
@@ -251,35 +246,24 @@ http.post(url: "https://foo.bar/baz", data: bytes(v: "body"))`
 
     it('can clone a task and edit it', () => {
       // clone a task
-      cy.getByTestID('task-card')
-        .trigger('mouseover')
-        .then(() => {
-          cy.get('.context-menu--container')
-            .eq(1)
-            .click()
-          cy.getByTestID('context-menu-item')
-            .contains('Clone')
-            .click()
-        })
+      cy.getByTestID('task-card').then(() => {
+        cy.getByTestID('context-menu-task').click()
+        cy.getByTestID('context-clone-task')
+          .click()
+          .type('{esc}')
+      })
 
       cy.getByTestID('task-card').should('have.length', 2)
 
       // assert the values of the task and change them
-      cy.getByTestID('task-card--name')
-        .eq(1)
-        .contains('🦄ask (clone 1)')
+      cy.getByTestID('task-card--name').contains('🦄ask (clone 1)')
 
-      cy.getByTestID('task-card')
-        .eq(1)
-        .trigger('mouseover')
-        .then(() => {
-          cy.getByTestID('context-cog-runs')
-            .eq(1)
-            .click()
-          cy.getByTestID('context-edit-task')
-            .eq(1)
```

Comment:
No longer needed because there is only 1 open popover in the DOM, and that is the one that was just clicked. Previously I guess there were multiple open popovers in the DOM and you needed to select a specific one,

Reply:
yeah this was due to an annotations bug actually. During the e2e tests for some reason _sometimes_ multiple "edit annotation" popups would appear, but only one would be visible. It was decided to work around the bug instead of fixing it.

Reply:
oh I spoke too soon - this is for Tasks. Nevermind - no idea what was going on here.

Reply:
>yeah this was due to an annotations bug actually. During the e2e tests for some reason sometimes multiple "edit annotation" popups would appear, but only one would be visible. It was decided to work around the bug instead of fixing it.

@genehynson do you have some evidence I haven't seen that this is an annotations bug and not a cypress bug? we've never been able to recreate it with annotations and it only happens in the firefox tests